util/junitxml/junitxml_test.go

```diff
@@ -0,0 +1,60 @@
+// Copyright 2021 Marty Pauley
```

Comment:
We set the header to "The prometheus authors" so we don't have to change on every edit